synapse/tools/autodoc.py

```diff
@@ -43,6 +43,9 @@
     'stortype',
     'bases',
     'custom',
+    'template',
+    'display',
+    'deprecated',
```

Comment:
The key `deprecated` should be covered in the `poptsToWords` map?

Reply:
`poptsToWords` is for props which are handled differently.